lib/elixir/lib/option_parser.ex

```diff
@@ -543,7 +542,12 @@ defmodule OptionParser do
         {[], false}
     end
 
-    {aliases, switches, strict?, allow_nonexistent_atoms?}
+    %{
+      aliases: opts[:aliases] || [],
+      allow_nonexistent_atoms?: opts[:allow_nonexistent_atoms] || false,
```

Comment:
Maybe we should start, since I think it's a bit pointless to *not* use it in key names. It serves the same purpose of signaling a boolean value, so I welcome it with open arms 😄